### PR TITLE
Add cketti as "speaker" for the Android workshop

### PR DESCRIPTION
**Pull request by cketti**

### CODE REVIEW

File: _data/speakers.yml [hunk 1]
```diff
@@ -362,3 +362,16 @@
     - {name: "twitter", link: "https://twitter.com/tbsprs"}
     - {name: "github", link: "https://github.com/johnjohndoe"}
     - {name: "google-plus", link: "https://plus.google.com/+tobiaspreuss"}
+-
+  id: 042
```
Comment (stefanhoth) [inline _data/speakers.yml:366]: Hacks!

Comment (cketti) [inline _data/speakers.yml:366]: I accept your compliment :)

**Comment by dlsniper** [comment]
Should I merge this?

**Comment by cketti** [comment]
I tested locally. Worked fine for me.

**Pull request merged**